packages/remix/src/index.server.ts

```diff
@@ -110,8 +110,10 @@ export * from '@sentry/node';
 
 export { captureRemixServerException, wrapRemixHandleError } from './utils/instrumentServer';
 export { ErrorBoundary, withErrorBoundary } from '@sentry/react';
+// eslint-disable-next-line deprecation/deprecation
 export { remixRouterInstrumentation, withSentry } from './client/performance';
 export { captureRemixErrorBoundaryError } from './client/errors';
+export { browserTracingIntegration } from './client/browserTracingIntegration';
```

Comment:
This should be exported from client, not server, I guess? 😅

Reply:
actually it is exported from client, but we should not export it here.

Reply:
We're sadly forced to export all client-side exports from the server-side as well, we previously had issues with unresolved imports and/or broken typings when we didn't.

Reply:
cc @Lms24  and/or @lforst you've done something like this for nextjs/sveltekit, can you point in the right direction here?

Reply:
Yeah we (unfortunately) export this on both sides, also in SvelteKit (and I believe NextJS). Kinda unfortunate but this is a limitation as long as we don't use sub-path/conditional package exports (or whatever they're called now).

![image](https://github.com/getsentry/sentry-javascript/assets/8420481/46b4c9b1-acf4-42d7-8189-44aa3b090f19)

Reply:
I wonder if it would not be better then to export the shim integrations as we do for the bundles - we already have that code for the bundles anyhow, and it's def. safer in regards to avoiding accidentally including browser-specific stuff..? that also takes care of warning if it is used etc..?

Reply:
We've had these discussions before and current status quo was to be mindful of where users are add the integration. Fwiw, the only common use case would be replay anyway. Only Remix requires users to additionally manually add `BrowserTracing` (something we should change long term btw). 

related
- https://github.com/getsentry/sentry-javascript/issues/9796
- https://github.com/getsentry/sentry-javascript/issues/9594

I guess shimming is not the end of the world but the problem is that we need to figure out where we draw the line.

Reply:
Also, i don't think shimming has high prio and we might solve this with the package exports anyway.

Reply:
👍 ok, so the outcome is we export browser tracing & replay in both client and server?